siikr/siikr_db_setup.sql

```diff
@@ -5,6 +5,10 @@
 -- Dumped from database version 14.10 (Ubuntu 14.10-0ubuntu0.22.04.1)
 -- Dumped by pg_dump version 14.10 (Ubuntu 14.10-0ubuntu0.22.04.1)
 
+-- Manually added, needed for healthchecks
+-- Should never ever have a password
+CREATE ROLE postgres WITH LOGIN SUPERUSER;
+
```

Comment:
We're going to have to add www-data here, I think.

Reply:
Or make it a script configured via secrets somehow.

Reply:
Yeah, this question has come up before. 

But now it occurs to me, does this docker composition approach even allow for connecting via unix sockets? 
Giving www-data this sort of permission is really only handy if they aren't going to be manually using credentials to get access via a host anyway.

Reply:
You're almost definitionally connecting between hosts.

Reply:
But that will still require some very annoying juggling with the setup scripts...hmm

Reply:
Don't get too involved with this just yet until I see how this works on the NAS that powers on but I've never actually connected to the internet before. 

If they have k8s, that's an init container or a helm manifest or something.

Reply:
I mean, it's a thing that needs doing regardless. 

I'd been writing most of the code under the assumption that no one was going to ask me to publish it, and it shows.